release_dashboard/lib/logic/error_to_string.dart

```diff
@@ -4,14 +4,14 @@
 
 import 'package:conductor_core/conductor_core.dart';
 
-/// Converts the error and stack trace objects to strings.
+/// Converts the error and stack trace objects to strings for display.
 String errorToString(Object error, StackTrace stackTrace) {
   final StringBuffer buffer = StringBuffer();
   if (error is ConductorException) {
     buffer.writeln('Conductor Exception:\n$error');
   } else {
     buffer.writeln('Error:\n$error');
   }
-  buffer.writeln('Stack Trace:\n$stackTrace');
+  buffer.writeln('\nStack Trace:\n$stackTrace');
```

Comment:
nit: Just write these as separate calls instead of adding `\n`

```
buffer.writeln('');
buffer.writeln('Stack Trace:');
buffer.writeln(stackTrace);
```

Or use a multiline string:

```
buffer.writeln('''
Stack Trace:
$stackTrace
''');
```